docs/modules/releasenotes/pages/whatsnew.adoc

```diff
@@ -10,13 +9,13 @@
 
 == Breaking Changes
 
-=== Running as Non-Root
+=== Running as non-root
 
 By default, OpenNMS now runs as the `opennms` user rather than `root`.
 
 WARNING: If you are upgrading from a version of OpenNMS Horizon older than 29, you will have to fix the ownership of your files.
 
-==== Fixing Permissions and Ownership
+==== Fixing permissions and ownership
 
 Since many OpenNMS instances are very large with thousands or even millions of RRD and similar data files in `$OPENNMS_HOME/share`,
```

Comment:
```suggestion
Since many {page-component-title} instances are very large with thousands or even millions of RRD and similar data files in `$\{OPENNMS_HOME}/share`,
```